tests/sqlparser_common.rs

```diff
@@ -3658,6 +3658,69 @@ fn parse_json_keyword() {
     );
 }
 
+#[test]
+fn parse_bignumeric_keyword() {
```

Comment:
@togami2864, can you please add tests for the resulting string?

Reply:
Sorry, what do you mean by 'resulting string'?

Reply:
After parsing a statement with this clause, what's the re-serialization of the query? (usually we use the `verified_stmt` or `one_parses_to` functions)